components/threading/channel_threads/thread_footer/index.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Creating a folder, normally is the approach that we follow for connected components, I'm not against using it here, but I want to be sure you did it on purpose. I think we can decide to follow this approach always, or not follow the approach at all, but having it sometimes and sometimes not can be a bit confusing. What do you think @deanwhillier? What do you think @calebroseland?

Reply:
Ok, I guess this is going to be eventually connected as least to pass the follow/unfollow actions, right?

Reply:
Lets wait until Dean give his opinion, probably we can create some help wanted tickets around moving existing components with sass file aside, into directory components.

Reply:
Sorry for the delay. We talked about this on my team/guild and decided to start putting single components in folders with index.js files whether connected or not.